src/Spryk/Model/Spryk/Executor/SprykExecutor.php

```diff
@@ -74,7 +74,6 @@ public function __construct(
         FileResolverInterface $fileResolver,
         FileDumperInterface $fileDumper
     ) {
-        $this->fileResolver = $fileResolver;
```

Comment:
Why did this get removed?

Reply:
It's a duplicate. It is initialized below.

Reply:
Ok